src/scss/components/ec-grid/_ec-grid.scss

```diff
@@ -0,0 +1,69 @@
+.ec-grid {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+
+  // If you don't want the outer columns to have margins then use .ec-grid--full-width
+  &--full-width {
```

Comment:
I prefer to fix this via an inner wrapper, and negative margins. Also, stylistic, use only one word, either `--full` or `--wide`